packages/admin-e2e-tests/src/utils/actions.ts

```diff
@@ -78,14 +78,17 @@ const verifyPublishAndTrash = async (
 	} );
 };
 
-const hasClass = async ( element: ElementHandle, elementClass: string ) => {
+const hasClass = async (
+	element: ElementHandle,
+	elementClass: string
+): Promise< boolean > => {
 	const classNameProp = await element.getProperty( 'className' );
 	const classNameValue = ( await classNameProp.jsonValue() ) as string;
 
 	return classNameValue.includes( elementClass );
 };
 
-const getInputValue = async ( selector: string ) => {
+const getInputValue = async ( selector: string ): Promise< unknown | null > => {
```

Comment:
I think this can be simplified to just `unknown` since it's a [top type](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#example-3)
```suggestion
const getInputValue = async ( selector: string ): Promise< unknown > => {
```